src/npm.js

```diff
@@ -14,11 +14,9 @@ export function info() {
   }));
 }
 
-const logWarning = ({ error, type, packages }) => {
```

Comment:
@Haroenv just a note that changes in this file are a bugfix not related to jsDelivr. This function expected `packages` to be an array but it was actually a string everywhere so it would fail on `packages.join()`